src/scrapers/pages/model.ts

```diff
@@ -277,6 +302,8 @@ function parseInfo($: CheerioAPI): ModelPage {
             || getAttribute<string>($('.socialList a:has(.amazonWLIcon)'), 'href'),
     }
 
+    const mostRecentVideos = parseVideoResult($, '.mostRecentPornstarVideos')
```

Comment:
`mostRecentVideos` should be `.pornstarUploadedVideos` ?
And we should also provide `taggedVideos` 🤔

Reply:
You are right. Model page does not have those different layouts.